### ISSUE TITLE
Output fields with longitude and latitude

### ISSUE THREAD

**Issue by kenkaigu** [DigitalCloud/nova-address-field#9]
Hello sir,
Can i use this package to show the output of a location on the map, if i already have the longitude and latitude coordinates...

**Comment by viktorivanov** [comment]
@kenkaigu, yes you can. Just use `initLocation` method on the AddressField. It takes two parameters: latitude and longitude.
